### DISCUSSION TITLE
System wide command_palette

### DISCUSSION THREAD

**Discussion by ksanderer** [TNorbury/command_palette#36] [Ideas]
Is it technically possible to expose the command_palette widget via a system-wide hotkey? This would enable a system-wide functionality similar to Alfred or [Raycast.io](https://www.raycast.com/).

I haven't researched the topic yet, but I thought maybe the author already has some insights into the problem.

**Comment by TNorbury** [comment]
I'll admit I haven't researched this either. The way that the package is set up now is just to be application wide. I imagine it's possible, most of the work would be on the native side I imagine.
